packages/flutter_tools/lib/src/dart/pub.dart

```diff
@@ -584,8 +588,10 @@ class _DefaultPub implements Pub {
       context: context,
       outputMode: outputMode,
     );
+
+    // TODO: probably can't remove this since non-get commands depend on it?
```

Comment:
which other commands?

Reply:
<img width="1224" alt="image" src="https://user-images.githubusercontent.com/159021/215602276-67cc2c46-2573-46ea-b4e3-6f7643d019f9.png">

I had checked the callers and saw multiple commands calling it.